nbs/api/14_quarto.ipynb

```diff
@@ -11,6 +11,14 @@
     "- order: 14"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        def __init__(self,path): self._path=path
</pre><p>Even although this is private a single line docstring saying what it's for might help people understand the code better.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Any preference on whether the docstring is for the class or the init method? </p>

Reply:
<p>Generally IIRC we put it on the class.</p>